lambdas/process-cri-callback/src/main/java/uk/gov/di/ipv/core/processcricallback/service/CriCheckingService.java

```diff
@@ -225,12 +225,11 @@ public JourneyResponse checkVcResponse(
             return ciMitUtilityService.getCiMitigationJourneyStep(cis).orElse(JOURNEY_FAIL_WITH_CI);
         }
 
-        if (!userIdentityService.areVCsCorrelated(
-                verifiableCredentialService.getVcStoreItems(clientOAuthSessionItem.getUserId()))) {
+        if (!userIdentityService.areVcsCorrelated(vcs)) { // can I just use the passed-in vcs here?
```

Comment:
Need to change this because of course we're checking if any of their VCs are uncorrelateed